### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by MischaBurgess**
Please check formatting, naming and logic :)
@michaelgeue
Thanks

### CODE REVIEW

File: Prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+def main():
+    # global name, email
+    name_from_email = {}  # create empty dictionary
+    email = input("Email: ")
+    while email != "":  # while email is not empty
+        name = email_to_name_function(email)  # get name from email using function
+        response = input("Is your name {}? (y/n) ".format(name))  # check name is correct
+        if response.lower() != "y" and response != "":  # is response is 'n'
```
Comment (michaelgeue) [inline Prac_05/emails.py:16]: Consider the user may possibly mean to press y, but misstyped.

File: Prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"""""
+Hex Colours task CP1404
+Mischa Burgess
+"""""
+
+Colours = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "azure1": "#f0ffff", "blue1": "#0000ff",
+           "BlueViolet": "#8a2be2", "chartreuse1": "#7fff00", "cyan2": "#00eeee", "DarkGreen": "#006400",
+           "DarkOrchid": "#9932cc", "DarkSalmon": "#e9967a"}  # create a hex colour dictionary
+
+Colour_choice = input("Enter the name of the colour you would like: ")
```
Comment (michaelgeue) [inline Prac_05/hex_colours.py]: lower case for variable name.

File: Prac_05/hex_colours.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"""""
+Hex Colours task CP1404
+Mischa Burgess
+"""""
+
+Colours = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "azure1": "#f0ffff", "blue1": "#0000ff",
```
Comment (michaelgeue) [inline Prac_05/hex_colours.py]: lower case variable name, or could be a constant.

File: Prac_05/state_names.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ")
+while state_code != "":  # while state code is not empty
+    if state_code.islower():  # checks if entered in lowercase
```
Comment (michaelgeue) [inline Prac_05/state_names.py]: consider changing the case when input, this will allow odd entries such as Tas

File: Prac_05/state_names.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        print(state_code, "is", CODE_TO_NAME[state_code])  # print description
+    else:
+        print("Invalid short state")
+    state_code = input("Enter short state: ")
+
+"""Part two"""
+
+print("*Part Two: print list of states*")
+for key, val in CODE_TO_NAME.items():  # for both key and values in dictionary
+    print("{:3} is {:8}".format(key, val))  # print with correct spacing
```
Comment (michaelgeue) [inline Prac_05/state_names.py:26]: 3 is good for formatting, but i'm unsure on what the 8 does.

File: Prac_05/emails.py (hunk 1 above)
Comment (MischaBurgess) [reply Prac_05/emails.py:16]: Good suggestion, thank you

File: Prac_05/hex_colours.py (hunk 2 above)
Comment (MischaBurgess) [reply Prac_05/hex_colours.py]: fixed :)

File: Prac_05/hex_colours.py (hunk 3 above)
Comment (MischaBurgess) [reply Prac_05/hex_colours.py]: fixed it up :)

File: Prac_05/state_names.py (hunk 5 above)
Comment (MischaBurgess) [reply Prac_05/state_names.py:26]: 8 is used for spacing the State/Territory full name :)

File: Prac_05/state_names.py (hunk 4 above)
Comment (MischaBurgess) [reply Prac_05/state_names.py]: updated it, thanks! :)

**Pull request merged**